tests/test_models.py

```diff
@@ -138,3 +142,51 @@ def test_build(simai_client):
     model_conf.pop("project_id")
 
     assert model_conf == MODEL_CONF_RAW
+
+
+def test_set_doa(simai_client):
+    """WHEN the Domain of Analysis is updated
+    THEN the simulation_volume is updated accordingly
+    """
+
+    model_conf = ModelConfiguration(project_id=MODEL_RAW["project_id"], **MODEL_CONF_RAW)
+
+    doa = model_conf.domain_of_analysis
+
+    new_height = {"position": "relative_to_center", "value": 0.5, "length": 15.2}
+
+    doa.Height = simai_client.models.doa_axis_definition(**new_height)
+
+    model_conf.domain_of_analysis = doa
+
+    assert model_conf.simulation_volume.get("Z").get("type") == new_height.get("position")
+    assert model_conf.simulation_volume.get("Z").get("value") == new_height.get("value")
+    assert model_conf.simulation_volume.get("Z").get("length") == new_height.get("length")
+
+
+def test_get_doa(simai_client):
+    """WHEN the Domain of Analysis is retrieved
+    THEN the params of the axes match
```

Comment:
parameters ?
. (period)

Reply:
Hello Romane. Files in the `tests` folders are not public, these comments are only meant for the developers to understand what the test does. As such we don't follow the same conventions there, for example there's no need to specify the parameters as these functions are never called by humans, it's all automated.

Reply:
Hi Marc, I understood that, it was just a suggestion (hence the question mark) to replace "params" with "parameters" and add periods at the end of sentences to clean up comments :) But if it seems too superficial, @yias feel free to ignore them.

Reply:
Thanks @romane-m for all the suggestions, they are very valuable :smile: